compiler/damlc/tests/daml-test-files/InterfaceGuardedNotExtended.daml

```diff
@@ -7,6 +7,7 @@
 
 module InterfaceGuardedNotExtended where
 
+import Daml.Script
```

Comment:
I think this will need to use the helper template trick, since we don't have guarded exercise on the ledger API

Reply:
Oh, I've been running these locally and it didn't fail, but looking at the code, it absolutely should - looking into this now

Reply:
Was a different issue, `InterfaceGuardedNotExtended.daml` isn't supposed to run, it shouldnt even have scripts in there, its purely to check that mention of exerciseGuarded without extended interfaces throws an error

Reply:
I'll update it to match `InterfaceGuarded` 1 to 1, with the error expectation